src/components/generateKeys/SecretKeyLogin.tsx

```diff
@@ -0,0 +1,132 @@
+import { ChangeEvent, useState } from 'react';
```

Comment:
If this is our login it shouldn't be named generateKeys, it should be just login because we can have more than 1 way to login in our account without generating keys like Stellar does.